{"content": {"suites": {"0": {"status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 0, "total_error": 2}, "tests": {"0": {"status": "ERROR", "message": "request = <SubRequest 'setup' for <Function test_home_page_title>>\n\n    @pytest.fixture()\n    def setup(request):\n        chrome_service = Service(ChromeDriverManager(chrome_type=ChromeType.GOOGLE).install())\n        chrome_options = Options()\n        options = [\n        \"--disable-gpu\",\n        \"--window-size=1920,1200\",\n        \"--ignore-certificate-errors\"\n    ]\n        for option in options:\n            chrome_options.add_argument(option)\n        request.cls.driver = webdriver.Chrome(service=chrome_service, options=chrome_options)\n        request.cls.util_obj =Utils(request.cls.driver)\n>       request.cls.locators_data=request.cls.util_obj.get_locators_data()\n\ntestCases\\conftest.py:23: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <utilities.utils.Utils object at 0x000002197E78E3D0>\n\n    def get_locators_data(self):\n        locator_path=os.path.join(\"locators.json\")\n>       file=open(locator_path)\nE       FileNotFoundError: [Errno 2] No such file or directory: 'locators.json'\n\nutilities\\utils.py:17: FileNotFoundError\n", "test_name": "test_home_page_title", "rerun": "0"}, "1": {"status": "ERROR", "message": "request = <SubRequest 'setup' for <Function test_sign_in_page_navigation>>\n\n    @pytest.fixture()\n    def setup(request):\n        chrome_service = Service(ChromeDriverManager(chrome_type=ChromeType.GOOGLE).install())\n        chrome_options = Options()\n        options = [\n        \"--disable-gpu\",\n        \"--window-size=1920,1200\",\n        \"--ignore-certificate-errors\"\n    ]\n        for option in options:\n            chrome_options.add_argument(option)\n        request.cls.driver = webdriver.Chrome(service=chrome_service, options=chrome_options)\n        request.cls.util_obj =Utils(request.cls.driver)\n>       request.cls.locators_data=request.cls.util_obj.get_locators_data()\n\ntestCases\\conftest.py:23: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <utilities.utils.Utils object at 0x000002197E825430>\n\n    def get_locators_data(self):\n        locator_path=os.path.join(\"locators.json\")\n>       file=open(locator_path)\nE       FileNotFoundError: [Errno 2] No such file or directory: 'locators.json'\n\nutilities\\utils.py:17: FileNotFoundError\n", "test_name": "test_sign_in_page_navigation", "rerun": "0"}}, "suite_name": "testCases/test_sigin_functionality.py"}}}, "date": "May 31, 2023", "start_time": 1685489611.9274719, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "0", "fail": "0", "skip": "0", "error": "2", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "2"}